service/resource/vaultcrt/create.go

```diff
@@ -96,9 +96,8 @@ func (r *Resource) ensureVaultRole(customObject certificatetpr.CustomObject) err
 			AllowSubdomains:  AllowSubDomains,
 			AltNames:         key.AltNames(customObject),
 			ID:               key.ClusterID(customObject),
-			Organizations: []string{
-				key.ClusterComponent(customObject),
-			},
+			Organizations: append([]string{key.ClusterComponent(customObject)},
+				key.Organizations(customObject)...),
```

Comment:
Above has to be done the same thing. It is not nice. There is an issue somewhere to make this more convenient.